### DISCUSSION TITLE
plugin tutorial doesn't work because stub plugin uses internal package

### DISCUSSION THREAD

**Discussion by jpicht** [99designs/gqlgen#2245] [Q&A]
### What happened?
I tried to follow https://gqlgen.com/reference/plugins/ to implement custom code generation, trying to base mine on [stubgen](https://github.com/99designs/gqlgen/tree/master/plugin/stubgen) as recommended.

Compiling fails with: `use of internal package github.com/99designs/gqlgen/internal/code not allowed`

### What did you expect?
The tutorial to work.

### Minimal graphql.schema and models to reproduce
Not applicable.

### versions
 - gqlgen v0.17.10
 - go version go1.18.3 linux/amd64

**Comment by jpicht** [comment]
I disagree with the classification as a discussion. It's either a documentation bug, or a bug in `stubgen`.

**Reply by frederikhors** [reply]
Please PR if is docs. Or wait for someone to help you.
